tests/unit/scripting.tcl

```diff
@@ -1578,3 +1578,69 @@ start_server {tags {"scripting"}} {
     }
 }
 
+# Cluster scripting tests
+
+# make sure the test infra won't use SELECT
+set old_singledb $::singledb
+set ::singledb 1
+
+start_server {overrides {cluster-enabled yes} tags {external:skip cluster}} {
+    r 0 cluster addslotsrange 0 16383
+    wait_for_condition 50 100 {
+        [csi 0 cluster_state] eq "ok"
+    } else {
+        fail "Cluster never became 'ok'"
+    }
```

Comment:
i'm not certain if we want this chunk of tests in cluster.tcl or in scripting.tcl.
on one hand, cluster.tcl was initially created in order to test modules outside the cluster test framework, and i'm not sure we wanna "pollute" other parts of the non-cluster redis tests with cluster related boilerplate.
on the other hand, these tests are more about scripting than cluster.
lastly, scripting.tcl is already huge, so maybe we wanna create script-cluster.tcl file?
or maybe in retrospect it was better to push all the flags and shebang tests in a script-flags.tcl file?
i don't have any concrete opinion... maybe i'd prefer cluster.tcl...

Reply:
I think script-cluster.tcl makes sense. In the long run, I think it will make sense to add extend specific tests so they cover cluster mode where necessary. I guess we'll just have some `start_cluster {}` fixture by then so it should also be cleaner.

Reply:
I did start working on a `start_cluster {}` fixture but haven't resolved the "single db" global stuff yet. I'm fine with a scripting-cluster.tcl file.